### PR TITLE
Shenyu issue3088

### PR DESCRIPTION
**Pull request by UniverseInHeart**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by midnight2104** [comment]
pls modify name of pr by https://shenyu.apache.org/community/issue-pr .

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 1]
```diff
@@ -159,24 +176,24 @@
...
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM rule_condition
+        WHERE id = #{id,jdbcType=VARCHAR}
     </delete>
 
     <delete id="deleteByRuleIds" parameterType="java.util.List">
-        delete from rule_condition
-        where rule_id in
+        DELETE FROM rule_condition
+        WHERE rule_id IN
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:189]: `WHERE` should be right-aligned with `FROM`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 2]
```diff
@@ -159,24 +176,24 @@
...
         <foreach item="ruleId" collection="list" open="(" separator="," close=")">
             #{ruleId, jdbcType=VARCHAR}
         </foreach>
     </delete>
 
     <delete id="deleteByQuery" parameterType="org.apache.shenyu.admin.model.query.RuleConditionQuery">
-        delete from rule_condition
-        where rule_id = #{ruleId,jdbcType=VARCHAR}
+        DELETE FROM rule_condition
+        WHERE rule_id = #{ruleId,jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:197]: `WHERE` should be right-aligned with `FROM`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 3]
```diff
@@ -123,19 +140,19 @@
...
-        set date_created = #{dateCreated,jdbcType=TIMESTAMP},
-        date_updated = #{dateUpdated,jdbcType=TIMESTAMP},
-        rule_id = #{ruleId,jdbcType=VARCHAR},
-        param_type = #{paramType,jdbcType=VARCHAR},
-        operator = #{operator,jdbcType=VARCHAR},
-        param_name = #{paramName,jdbcType=VARCHAR},
-        param_value = #{paramValue,jdbcType=VARCHAR}
-        where id = #{id,jdbcType=VARCHAR}
+        UPDATE rule_condition
+        SET date_created = #{dateCreated,jdbcType=TIMESTAMP},
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:144]: `SET` should be right-aligned with `UPDATE`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 4]
```diff
@@ -30,48 +30,65 @@
...
-        #{paramName,jdbcType=VARCHAR}, #{paramValue,jdbcType=VARCHAR})
+        INSERT INTO rule_condition (id,
+                                    date_created,
+                                    date_updated,
+                                    rule_id,
+                                    param_type,
+                                    operator,
+                                    param_name,
+                                    param_value)
+        VALUES (#{id,jdbcType=VARCHAR},
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:80]: `VALUES` should be right-aligned with `INTO`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 5]
```diff
@@ -30,48 +30,65 @@
...
     </select>
 
     <select id="selectByRuleIdSet" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from rule_condition
-        where rule_id in
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM rule_condition
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:63]: `FROM` should be right-aligned with `SELECT`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 6]
```diff
@@ -30,48 +30,65 @@
...
     </select>
 
     <select id="selectByQuery" parameterType="org.apache.shenyu.admin.model.query.RuleConditionQuery"
             resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from rule_condition
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM rule_condition
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:54]: `FROM` should be right-aligned with `SELECT`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml [hunk 7]
```diff
@@ -30,48 +30,65 @@
...
     </sql>
 
     <select id="selectById" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from rule_condition
-        where id = #{id,jdbcType=VARCHAR}
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM rule_condition
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:46]: `FROM` should be right-aligned with `SELECT`

File: shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml (hunk 1 above)
Comment (UniverseInHeart) [reply shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml:189]: copy

> pls modify name of pr by https://shenyu.apache.org/community/issue-pr . input about #3088 in your comment

copy that

**Pull request closed without merging**